CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ created.
   GPU utilization via Lightning's `GpuStatsMonitor`, switch `monitor_loading` to check batch loading times via
   `BatchTimeCallback`, and `pl_profiler` to turn on the Lightning profiler (`simple`, `advanced`, or `pytorch`)
 - ([#544](https://github.com/microsoft/InnerEye-DeepLearning/pull/544)) Add documentation for segmentation model evaluation.
+- ([#637](https://github.com/microsoft/InnerEye-DeepLearning/pull/637)) Add option to encode in chunks and to load pre-cached dataset to the histo pipeline.
```

Comment:
Minor suggestion to clarify that the "option [...] to load pre-cached dataset" refers to CPU vs GPU memory.